pkg/apis/internal.acorn.io/v1/appspec.go

```diff
@@ -365,6 +365,9 @@ type Container struct {
 	ComputeClass *string                `json:"class,omitempty"`
 	Memory       *int64                 `json:"memory,omitempty"`
 
+	// Metrics is available on containers and jobs, but not sidecars
```

Comment:
Yep. The AML scheme will not allow users to specify metrics in a sidecar. It has to be done at the container level.

If the metrics are available from the sidecar, it will still work as long as the port that they specify for the metrics is one that the sidecar is using.

Reply:
Nice! Sounds good to me.